src/vcr_pytest.py

```diff
@@ -0,0 +1,62 @@
+import os
```

Comment:
Do we need this code at all here?

Reply:
I was wondering this as well

Reply:
Team discussion - let's get rid of this, but capture key lines of code like VCR().register_serializer( our_cleaner_serializer )